android/library/src/main/java/io/logto/android/Logto.kt

```diff
@@ -1,9 +1,19 @@
 package io.logto.android
 
+import android.content.Context
```

Comment:
the path should be `src/main/kotlin`?

Reply:
I follow the way google dose in all open-source project written in pure kotlin.
Use the default way: [source organization](https://developer.android.com/kotlin/add-kotlin#source) to avoid sourceSet setting

Reply:
fine